react/01.js

```diff
@@ -0,0 +1,57 @@
+
+class  TodoApp extends React.Component {
+  constructor() {
+    super()
+    this.state={
+      todoList: [],
+      value: "",
+    }
+  }
+
+
+  onChange(e) {
+    this.setState({value: e.target.value})
+  }
+
+  add() {
+    let arr = this.state.todoList;
+    arr.push({'comment':this.state.value,'status':1});
```

Comment:
json形式ではなく普通のオブジェクト形式で記述する方が一般的です。
また、スペースも以下のように付けるとより現場に近いです。
```suggestion
    arr.push({ comment: this.state.value, status: 1 });
```